scripts/PyChop/ISISDisk.py

```diff
@@ -178,7 +179,7 @@ def __LETgetResolution(self, single_mode, Etrans=None, Ei_in=None):
             ie_list = np.where(np.abs(np.array(Eis)-Ei) == np.min(np.abs(np.array(Eis)-Ei)))[0]
             Et = np.linspace(0.05, 0.95*Ei, 19, endpoint=True) if (Etrans is None) else Etrans
         else:
-            ie_list = range(len(Eis))
+            ie_list = list(range(len(Eis)))
```

Comment:
does not need to be a list

However please modify  lines   203-206 to:     
```
if len(ie_list) == 1:
            chop_width = chop_width[ie_list[0]]
            mod_width = mod_width[ie_list[0]]
            res_el = res_el[ie_list[0]]
```
Only the last one will complain